File: spec/code_samples/Python_requests/list@cleaned@{list_name}/get.py
import requests

url = "https://cleanse.studio/api/v1/list/cleaned/%7Blist_name%7D"

querystring = {"username":"SOME_STRING_VALUE"}

headers = {'content-type': 'application/json'}

response = requests.request("GET", url, headers=headers, params=querystring)

print(response.text)

File: spec/code_samples/Node_request/lists@cleaned/get.js
var request = require("request");

var options = { method: 'GET',
  url: 'https://cleanse.studio/api/v1/lists/cleaned',
  qs: { username: 'SOME_STRING_VALUE' },
  headers: { 'content-type': 'application/json' } };

request(options, function (error, response, body) {
  if (error) throw new Error(error);

  console.log(body);
});
